CHANGELOG.md

```diff
@@ -57,6 +57,40 @@ Swift 5.0
 Swift 4.2
 ---------
 
+* [SE-0185][]
+
+  Protocol conformances can now be synthesized in extensions in the same file as
```

Comment:
"can now be" -> "are now"

?

Reply:
I think "can" is right here: the synthesis doesn't always happen.